### PR TITLE
Create json data import description

### PR DESCRIPTION
**Pull request by scottwittenburg**
Provides a more flexible approach to describe datasets to import.

### CODE REVIEW

**Comment by scottwittenburg** [comment]
FYI @curtislisle @dzenanz

File: server/miqa_server/session.py [hunk 1]
```diff
@@ -116,11 +133,16 @@ def csvImport(self, params):
...
-                    currentAssetstore, parent=scanFolder, parentType='folder', params={
-                        'fileIncludeRegex': '.+[.]nii[.]gz$',
-                        'importPath': niftiFolder,
-                    }, progress=noProgress, user=user, leafFoldersAsItems=False)
+                scanImages = scan['images']
+                for scanImage in scanImages:
+                    absImagePath = os.path.join(niftiFolder, scanImage)
+                    imageItem = Item().createItem(name=scanImage, creator=user, folder=scanFolder, reuseExisting=True)
+                    Assetstore().importFile(imageItem, absImagePath, user, name=scanImage)
+                # Assetstore().importData(
```
Comment (dzenanz) [inline server/miqa_server/session.py]: Was this commented out on purpose? Why not just delete it so it is clear the above code fragment is its replacement?

File: server/miqa_server/session.py [hunk 2]
```diff
@@ -136,13 +136,14 @@ def dataImport(self, params):
...
-                    Assetstore().importFile(imageItem, absImagePath, user, name=scanImage)
-                # Assetstore().importData(
-                #     currentAssetstore, parent=scanFolder, parentType='folder', params={
-                #         'fileIncludeRegex': '.+[.]nii[.]gz$',
-                #         'importPath': niftiFolder,
-                #     }, progress=noProgress, user=user, leafFoldersAsItems=False)
+                    # imageItem = Item().createItem(name=scanImage, creator=user, folder=scanFolder, reuseExisting=True)
+                    # Assetstore().importFile(imageItem, absImagePath, user, name=scanImage)
+                    logger.info('importing next file {0}'.format(scanImage))
+                    Assetstore().importData(
```
Comment (dzenanz) [inline server/miqa_server/session.py:184]: Oh, I see, reviving it.

**Review by dzenanz — APPROVED** [review]
Judging by your description today, this is a good way to import big archives, and is a reasonable way to automate import via a tool. Some nitpicking inline.

**Pull request merged**